python/cuml/test/dask/test_random_forest.py

```diff
@@ -334,14 +335,67 @@ def test_rf_regression_dask_cpu(partitions_per_worker, cluster):
         X_train_df, y_train_df = dask_utils.persist_across_workers(
             c, [X_train_df, y_train_df], workers=workers)
 
-        cu_rf_mg = cuRFR_mg(**cu_rf_params)
-        cu_rf_mg.fit(X_train_df, y_train_df)
+        cuml_mod = cuRFR_mg(**cu_rf_params)
+        cuml_mod.fit(X_train_df, y_train_df)
 
-        cu_rf_mg_predict = cu_rf_mg.predict(X_test, predict_model='CPU')
+        cuml_mod_predict = cuml_mod.predict(X_test, predict_model='CPU')
 
-        acc_score = r2_score(cu_rf_mg_predict, y_test)
+        acc_score = r2_score(cuml_mod_predict, y_test)
 
         assert acc_score >= 0.67
 
     finally:
         c.close()
+
+
+@pytest.mark.parametrize('partitions_per_worker', [1])
+@pytest.mark.parametrize('output_class', [True, False])
+def test_rf_classification_dask_fil_predict_proba(partitions_per_worker,
+                                                  cluster,
+                                                  output_class):
+
+    # Use CUDA_VISIBLE_DEVICES to control the number of workers
+    c = Client(threads_per_worker=1, n_workers=1)
+
+    try:
+
+        X, y = make_classification(n_samples=10000, n_features=30,
```

Comment:
How long does this test take to run? let's make sure we're not adding any more multi-minute tests if we could shrink the datasets and still achieve the desired outcome.

Reply:
6.7 seconds each . The overall test function takes 13.4 seconds

Reply:
reduced it down to a single test, it takes 6.5s to run and 1.7s to teardown

Reply:
also the time required to run the function is:
```
1.78s teardown cuml/test/dask/test_random_forest.py::test_rf_classification_dask_fil_predict_proba[5]
1.47s call     cuml/test/dask/test_random_forest.py::test_rf_classification_dask_fil_predict_proba[5]
```